src/commands/site-cli.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tribe\Test;
+
+if ( $is_help ) {
+	echo "Waits for WordPress to be correctly set up to run a wp-cli command in the stack.\n";
+	echo PHP_EOL;
+	echo colorize( "signature: <light_cyan>{$cli_name} site-cli [ssh] [...<commands>]</light_cyan>\n" );
+	echo colorize( "example: <light_cyan>{$cli_name} site-cli plugin list --status=active</light_cyan>\n" );
+	echo colorize( "example: <light_cyan>{$cli_name} site-cli theme install twentytwenty</light_cyan>\n" );
+	echo colorize( "example: <light_cyan>{$cli_name} site-cli _install</light_cyan>" );
+
+	return;
+}
+
+setup_id();
+$command = $args( '...' );
+
+if ( 'wp' === reset( $command ) ) {
+	// If there's an initial `wp` remove it; the user might have called the command with `tric site-cli wp ...`.
+	array_shift( $command );
+}
+
+/*
+ * wp-cli already comes with a `shell` command that will open a PHP shell, same as `php -a`, in it.
+ * As much as it would be ideal to use the `shell` sub-command to open a shell... we cannot use the `shell` word.
+ */
+$open_bash_shell = reset( $command ) === 'bash';
+
+/*
+ * This is an "internal" shortcut to quickly install WordPress for the purpose of performing some operations on its
+ * structure.
+ */
+$_install = reset( $command ) === '_install';
```

Comment:
I don't love _install but perhaps I'm just misguided. Other ideas:

- setup
- init
- install (we can still detect the usage here and strip it)
- auto-install
- default-install
- simple-install

Reply:
The idea here would be to prefix it with `_` to denote an internal command.
The `_install` shortcut is internal as it comes with pre-wired parameters that might not be the ones used by the various test installations.
I had, initially, tried `quick-install` but felt like that was misleading.

I'm really open to suggestions here.

Reply:
The `_` prefix always feels like a janky dev convention, but I understand what you are aiming to do. I don't have any real strong arguments against it other than my own personal preference...which is just an opinion that I don't feel passionate about. If you like `_install`, let's keep it. :handshake: